tv-shows-app/src/app/all-shows/[id]/page.tsx

```diff
@@ -0,0 +1,18 @@
+'use client'
+
+import { LayoutWithSidebar } from "@/components/shared/Layouts/LayoutWithSidebar/LayoutWithSidebar";
+import { useParams } from "next/navigation";
+import { ShowReviewSection } from "@/components/features/shows/ShowReviewSection/ShowReviewSection";
+import { AuthRedirect } from "@/components/shared/AuthRedirect/AuthRedirect";
+
+export default function ShowItem() {
+	const params = useParams();
+
+	return (
+		<AuthRedirect>
+			<LayoutWithSidebar>
```

Comment:
Ovo nam je zasad okej, vidjet ces bonus zadatak u novoj zadaći kako rješit ovaj dio bez da wrappas svaki page zasebno.
-> nextjs route groups